domain/src/main/java/ru/startandroid/domain/models/Astro.kt

```diff
@@ -0,0 +1,21 @@
+package ru.startandroid.domain.models
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+class Astro {
```

Comment:
Let's convert all these API data classes to Kotlin Data Classes. Just add "data" word before "class" and move all parameters to constructor.